fairlearn/reductions/_grid_search/__init__.py

```diff
@@ -4,9 +4,10 @@
 """Package for Grid Search."""
 
 from .grid_search_result import GridSearchResult  # noqa: F401
-from .grid_search import GridSearch  # noqa: F401
+from .grid_search import GridSearch, _GridGenerator  # noqa: F401
 
 __all__ = [
+    "_GridGenerator",
```

Comment:
Why?

Reply:
Couldn't use it in test otherwise for some reason :-| I'll see how I can get rid of it.

Reply:
Fixed that now. Not sure what happened the previous time.